### ISSUE TITLE
ASP.NET Beta8: Unit Test project

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#389]
Review and update for `beta8` if applicable.

[Sayed]: Side note, when we get the xUnit template we should also send a PR to SideWaffle to update the [xUnit template](https://github.com/ligershark/side-waffle/tree/master/Project%20Templates/xUnitTestProject)

**Comment by peterblazejewicz** [comment]
@bradwilson @sayedihashimi 
are we OK to update xUnit template for beta8?
I could see some work on this:
https://github.com/xunit/dnx.xunit/commit/7784d7b55df05652b4ca6a72e47c1ec5bafd74c7
https://github.com/xunit/dnx.xunit/commit/eb02b4709e0b038edef1c7da52e812173fbe0018
https://github.com/xunit/dnx.xunit/blob/master/test/test.xunit.runner.dnx/project.json

It appears to work with:
```
  "dependencies": {
    "xunit": "2.1.0-*",
    "xunit.runner.dnx": "2.1.0-*"
  },
  "commands": {
    "test": "xunit.runner.dnx"
  },

  "frameworks" : {
    "dnx451": { },
    "dnxcore50" : { }
  }
```
```
dnx test
xUnit.net DNX Runner (64-bit DNX 4.5.1)
  Discovering: UnitTest
  Discovered:  UnitTest
  Starting:    UnitTest
  Finished:    UnitTest
=== TEST EXECUTION SUMMARY ===
   UnitTest  Total: 1, Errors: 0, Failed: 0, Skipped: 0, Time: 0.161s
```
?

**Comment by bradwilson** [comment]
Beta 8 has not shipped yet, so no.

The officially supported versions are listed here: http://xunit.github.io/docs/getting-started-dnx.html

I would use *only* those explicit versions, *not* wildcards.

**Comment by peterblazejewicz** [comment]
Thanks!
@sayedihashimi ?

**Comment by sayedihashimi** [comment]
Thanks. I think we can release beta8 initially and keep the existing content. Then when xunit is updated we can re-release.

**Comment by peterblazejewicz** [comment]
@sayedihashimi 
I've modified list in #386  to reflect this, but left `beta8` label for reference

**Comment by sayedihashimi** [comment]
Should be fixed in `0.0.81` closing.
